src/main/java/com/tg5/Application.java

```diff
@@ -1,11 +1,17 @@
 package com.tg5;
 
+import com.tg5.service.MessageConsumer;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.scheduling.annotation.EnableScheduling;
 
 @SpringBootApplication(scanBasePackages = {"edu.miu.common", "com.tg5"})
+@EnableScheduling
 public class Application {
 
+	@Autowired
```

Comment:
u sure you need this here?

Reply:
We need this for creating the consumer Bean.